lib/Doctrine/ORM/Tools/Pagination/WhereInWalker.php

```diff
@@ -96,10 +95,7 @@ public function walkSelectStatement(SelectStatement $AST)
                         ),
                     ]
                 );
-            } elseif (
-                $AST->whereClause->conditionalExpression instanceof ConditionalExpression
-                || $AST->whereClause->conditionalExpression instanceof ConditionalFactor
-            ) {
```

Comment:
Was this `elseif` superfluous that it could be changed into an `else`.

Reply:
I believe so, yes.